### PR TITLE
Task #143449 chore: support HTTP_X_IDENTIFIER in get request of users API

### PR DESCRIPTION
**Pull request by shraddhaSankpal27**

### CODE REVIEW

File: src/users/user.php [hunk 1]
```diff
@@ -156,14 +156,23 @@ public function get()
 	{
 		$input = JFactory::getApplication()->input;
 		$id = $input->get('id', 0, 'int');
+		$xidentifier	= $input->server->get('HTTP_IDENTIFIER', '', 'String');
```
Comment (coolbung) [inline src/users/user.php]: The header needs to be `X-Identifier`. Please change that in the user update/save method also.

Comment (shraddhaSankpal27) [reply src/users/user.php]: @coolbung Updated Please check

File: src/users/user.php [hunk 2]
```diff
@@ -156,14 +156,23 @@ public function get()
...
+		$xidentifier	= $input->server->get('HTTP_X_IDENTIFIER', '', 'String');
 
 		/*
 		 * If we have an id try to fetch the user
 		 * @TODO write user field mapping logic here
 		 */
 		if ($id)
 		{
-			$user = JUser::getInstance($id);
+			if ($xidentifier)
```
Comment (coolbung) [inline src/users/user.php]: This if/else is not needed. Check `retriveUser` it is already taking into account id/email/username.

Comment (shraddhaSankpal27) [reply src/users/user.php]: @coolbung done changes

**Comment by coolbung** [comment]
LGTM. @manojLondhe @thite-amol you have any feedback ?

**Pull request merged**